lib/metasurfaces/b30_library_siz_auto.m

```diff
@@ -0,0 +1,39 @@
+function [m,n,m1]=b30_library_siz_auto(phi,n_lam)
```

Comment:
Could you add a short explanation of what this function does and when it should be used?